src/main/java/core/basesyntax/FileWork.java

```diff
@@ -8,7 +15,29 @@
  * Результат: web wide width world</p>
  */
 public class FileWork {
+    private static final String WORD_STARTS_WITH = "w";
+
     public String[] readFromFile(String fileName) {
-        return null;
+        Path inputFilePath = Paths.get(fileName);
+        ArrayList<String> inputFile = new ArrayList<>();
```

Comment:
in this case a good practice is to use type of a parent
```suggestion
        List<String> inputFile = new ArrayList<>();
```

Reply:
fixed